docs/roadmap.rst

```diff
@@ -1,70 +1,127 @@
 .. include:: links.rst
 
---------------------
-Development road map
---------------------
+Development road-map
+====================
 This road-map serves as a guide for developers as well as a way for us to
-communicate to users and other stake-holders aboout the expectations they should
-have about the current functionality of the software and future developments.
+communicate to users and other stake-holders about the expectations they should
+have for the current functionality of the software and future developments.
 
-Version 0.3 (Targetted for March 1st, 2020)
--------------------------------------------
-This version should be considered an early alpha of the software, but will
-contain a full pipeline of processing from a raw BIDS dataset to analyzable data.
+If you would like to be part of the team developing this road-map, please be sure to
+read our `Contributors Guidelines <https://www.nipreps.org/community/CONTRIBUTING>`__.
+Then, please write one of the developers to be invited to our bi-weekly meetings.
 
-At this point, the processing pipeline will include the following major steps:
+This road-map proposes a :abbr:`RERO (release early, release often)` philosophy, scheduling
+a monthly release until the first stable 1.0 release is reached.
 
-#. Susceptibility distortion correction.
-    Using `SDCFlows <https://github.com/poldracklab/sdcflows>`__
+.. important::
 
-#. Signal drift estimation
-    Leveraging the :math:`b=0` extraction, rescaling and averaging that was merged in `#50 <https://github.com/nipreps/dmriprep/pull/50>`__
+    Updated: Dec 18, 2020
+    Latest release: `0.3.0 (October 13, 2020) <changes.html#october-13-2020>`__.
 
-Version 0.4 (April 1st, 2020)
------------------------------
+Version 0.4 (Before end of 2020)
+--------------------------------
+Version 0.4 will condense all the outcomes of our sprint towards ISMRM's 2021 abstracts deadline.
+This mostly includes house-keeping work, and most prominently, the integration of the *SDCFlows* 2.0
+alpha releases, which makes *dMRIPrep* go ahead of *fMRIPrep* in addressing distortions caused by
+:math:`B_0` inhomogeneity.
+
+This release will also include Salim's efforts in `#144 <https://github.com/nipreps/dmriprep/pull/144>`__
+to provide a temporary implementation of head-motion and eddy-currents correction using
+FSL's ``eddy``.
+This temporary solution will be replaced by our 3dSHORE-based algorithm ported from QSIPREP,
```

Comment:
Can we go ahead and name it yet? I thought Eddy Motion Correction is pretty good...